<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Segmentationwidget</class>
 <widget class="QWidget" name="Segmentationwidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>282</width>
    <height>319</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>282</width>
    <height>319</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>345</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>144</height>
      </size>
     </property>
     <property name="title">
      <string>Seeds and Mask</string>
     </property>
     <widget class="QPushButton" name="ClearButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>250</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>27</height>
       </size>
      </property>
      <property name="text">
       <string>Clear Seeds</string>
      </property>
     </widget>
     <widget class="QSplitter" name="splitter">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>250</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>27</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="eraserButton">
       <property name="text">
        <string>Erase</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="drawButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>Draw</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="autoExclusive">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>250</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>27</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QPushButton" name="pushButtonShowSeeds">
       <property name="text">
        <string>View Seeds</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonShowMask">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>23</height>
        </size>
       </property>
       <property name="text">
        <string>View Mask</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="SegmentationButton">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>253</width>
        <height>23</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>23</height>
       </size>
      </property>
      <property name="text">
       <string>Start Segmentation</string>
      </property>
     </widget>
     <zorder>ClearButton</zorder>
     <zorder>splitter</zorder>
     <zorder>splitter</zorder>
     <zorder>splitter_3</zorder>
     <zorder>SegmentationButton</zorder>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="groupBoxPathFunction">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>177</height>
      </size>
     </property>
     <property name="title">
      <string>Path Propagation Function</string>
     </property>
     <widget class="QSplitter" name="splitter_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>321</width>
        <height>81</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>81</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QRadioButton" name="pfmaxgeo">
       <property name="text">
        <string>MaxFunction with Geodesic Restriction</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="pfmax">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>MaxFunction</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="pfsum">
       <property name="text">
        <string>SumFunction</string>
       </property>
      </widget>
     </widget>
     <widget class="QSplitter" name="splitter_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>100</y>
        <width>231</width>
        <height>27</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>27</height>
       </size>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QLabel" name="label">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Alpha:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Beta:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>27</height>
        </size>
       </property>
       <property name="text">
        <string>Thickness</string>
       </property>
      </widget>
     </widget>
     <widget class="QDoubleSpinBox" name="AlphaSpinBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>130</y>
        <width>71</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="minimum">
       <double>-1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>1.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="BetaSpinBox">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>61</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="maximum">
       <double>4.000000000000000</double>
      </property>
      <property name="singleStep">
       <double>0.100000000000000</double>
      </property>
      <property name="value">
       <double>0.500000000000000</double>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="thickDoubleSpinBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>24</height>
       </size>
      </property>
      <property name="decimals">
       <number>1</number>
      </property>
      <property name="singleStep">
       <double>1.000000000000000</double>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
